### PR TITLE
Fixed search and portfolio form

### PR DESCRIPTION
**Pull request by oerkperl**

### CODE REVIEW

File: app/components/portfolio/FormModal.tsx [hunk 1]
```diff
@@ -121,7 +123,9 @@ export const FormModal: React.FC<ModalProps> = ({ onClose }) => {
                       Cancel
                     </button>
                     <button
-                      className=" w-1/2 h-12 rounded-lg bg-white dark:bg-gray-800 hover:bg-indigo-600 hover:text-white"
+                      className={`w-1/2 h-12 rounded-lg bg-white dark:bg-gray-800  hover:text-white ${
+                        canAddAsset ? "hover:bg-green-600" : "hover:bg-gray-600"
+                      }`}
                       disabled={!canAddAsset}
                       onClick={() => {
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/FormModal.tsx]: onClick={addNewAsset}

File: app/components/portfolio/MyAsset.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ export const MyAsset: React.FC<{
 
         {hasError && (
           <button
-            className="px-1 rounded-md hover:bg-indigo-600"
+            className="px-1 rounded-md hover:bg-indigo-600 hover:text-white"
             onClick={() => {
               fetchAsset();
             }}
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/MyAsset.tsx]: onClick={addNewAsset}

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/components/portfolio/FormModal.tsx (hunk 1 above)
Comment (oerkperl) [reply app/components/portfolio/FormModal.tsx]: Trying to disable the button if form not filled properly(according to the requirment). I do not undertand the issue you are pointing out.

Comment (oerkperl) [reply app/components/portfolio/FormModal.tsx]: I get what you say now.

File: app/components/portfolio/MyAsset.tsx (hunk 2 above)
Comment (oerkperl) [reply app/components/portfolio/MyAsset.tsx]: Ok!

**Pull request merged**